data/categories.txt

```diff
@@ -2891,7 +2891,7 @@ sk:3Piknik
 sw:Sehemu ya madhazi
 fa:مکان پیک نیک
 
-amenity-place_of_worship|@tourism
+amenity-place_of_worship
```

Comment:
@vng do we have a way/type to quickly search for all objects that have Wikipedia tag?

@RedAuburn can you please analyze existing OSM features with wikipedia/wikidata tags and recommend a way/algo to highlight sightseeing based on it? Or are there any other better ideas?

Reply:
Can implement something like: ?wiki
Anyway should iterate all features here.

Reply:
Can you please test if it's slow or not? We can add it to a clickable search category (without a question is better).

Reply:
A Wikipedia page is a good indication of relevance, but in old cities lots of things have a Wikipedia page just because they're old. Wikidata objects can be made for anything so aren't a good indicator. 
Maybe we could query and store the yearly Wikipedia pageviews as a vague popularity index?: https://wikitech.wikimedia.org/wiki/Analytics/AQS/Pageviews

Reply:
We should include some important temples in sightseeing somehow. Think Sagrada de Familia in Barcelona, for example. Can you please create a separate issue?